package.json

```diff
@@ -96,6 +96,7 @@
     "browserslist": "^4.16.3",
     "caniuse-lite": "^1.0.30001204",
     "circular-dependency-plugin": "^5.2.1",
+    "codecov": "^3.8.1",
```

Comment:
I'd do a spot check that this module doesn't just do the same thing. This was the case on ICU4X's codecov github action implementation.

Reply:
yeah, I had a look at their implementation, and they do the actual work, they do not just fetch something from the internet and run it.
And that's why the behavior is different too :(